composer.json

```diff
@@ -9,11 +9,12 @@
             "email": "guiwoda@gmail.com"
         }
     ],
-    "require": {},
+    "require": {
+    },
     "require-dev": {
         "phpunit/phpunit": "^5.0",
         "php-vcr/php-vcr": "^1.2",
-        "friendsofphp/php-cs-fixer": "^1.12"
+        "friendsofphp/php-cs-fixer": "^2.13"
```

Comment:
Same applies here.

Reply:
Can not launch PHP-cs-fix on CentOS 7 with PHP 7.2. So I updated on the latest version. I still had a problem finding the class Finder, so I installed a fix on the Friendsofphp / php-cs-fix repository. I can revert the changes if necessary.